src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1SpaceBookingTest.kt

```diff
@@ -657,6 +681,51 @@ class Cas1SpaceBookingTest {
       assertThat(response[4].tier).isEqualTo("A")
       assertThat(response[4].person.crn).isEqualTo("CRN_CURRENT1")
     }
+
+    @Order(9999)
+    @ParameterizedTest
+    @MethodSource("spaceBookingSummaryStatusCases")
+    fun `Search generates the correct statuses`(
+      testCaseForSpaceBookingSummaryStatus: TestCaseForSpaceBookingSummaryStatus,
+      expectedResultStatus: Cas1SpaceBookingSummaryStatus,
+      testCaseIndex: Int,
+    ) {
+      val (_, jwt) = `Given a User`(roles = listOf(CAS1_FUTURE_MANAGER))
+
+      val crnForStatusTest = "STATUS_TEST_BOOKING__$testCaseIndex"
```

Comment:
I think this will lead to the filtered list of results getting more than 1 result (in the cases where we test the boundaries for a specific status. For example, we test the `arrivingIn2Weeks` status 14 times to cover the 14 days in the period where we service up this status.
Let me know if I'm missing something :)